src/js/inputs/text.js

```diff
@@ -15,6 +15,7 @@
           '<div id="error-text" class="error-text"></div>' +
         ''),
         events:{
+          'change input': 'keyPress',
```

Comment:
is it possible to make some of these optional events?  Seems like it would be more efficient to only have them connected when necessary or desired